### ISSUE TITLE
Track URL ?

### ISSUE THREAD

**Issue by tmeedend** [JustaPenguin/assetto-server-manager#1025]
Hi, I see in the Content > Tracks that we can fill an URL of the Track. It says "Download links for tracks will be shown in Content Manager's server information screen.". So I have filled this field but I don't see anything in Content Manager when I select my server. Where is this displayed ? It would be very helpful to resolve checksum issues.

Thank you.

**Comment by cjoneill** [comment]
@tmeedend see if this info helps. You probably haven't enabled the right option / opened the port on your server firewall.

https://github.com/JustaPenguin/assetto-server-manager/wiki/Sharing-Content-With-Content-Manager-Wrapper

**Comment by tmeedend** [comment]
Thank you I understand better that's perfect it works ! I did no't know this.

One last thing, is it possible to make real links in the webapp ? It seems to be done with javascript and we cannot open links in new tabs. I woulk like to open all my tracks in new tabs to update their URL but I cannot, I have to manually open each one in the same window, and it takes time to load. A middle click to open on new tab would help a lot. 

Thank you :)

**Comment by cjoneill** [comment]
I agree that would be cool, but I'd suggest opening a feature request as a new issue

**Comment by tmeedend** [comment]
Hi, ok I will do it. 

One last thing about this url field, sometimes in the ui_track.json file of a track (and possibly of cars too), there is already an 'url' field with the url of the track (near the version). But I can see that it's not read by server manager ? Would it be possible ? What do you think ? For the moment, I see that if I fill the URL, it's written in another file (meta_data.json) with the notes. Why is this done in another file ?

And one last las idea, what do you think of the possibility to generate an archive of tracks / cars deployed so they can be downloaded ? This way, we would not need an external url at all. Is this a problem with permissions to redistribute the track ?

Thanks !
